### ISSUE TITLE
Collect data between redirects

### ISSUE THREAD

**Issue by KES777** [miyagawa/Plack-Middleware-Debug#30]
The problem I have run into with Plack::Middleware::Debug::Log4perl

The log messages are not displayed that were logged while processing POST data, when I POST to page that redirect (302) back to page with this form 
When redirect occours the collected data in DebugPanel must be saved somewhere (maybe in session???)  and displayed them on redirecting page.

But this is applyed also to 'Request variables', 'Response', 'Environment', 'Memory', 'Timer', etc.
They must show values for those two requests
1. POST. When I try to save form
2. GET. When form is saved and result page is showed

**Comment by miyagawa** [comment]
I agree this is a nice addition, but you should better refrain from using the word "must" in such a way you demand the feature to be implemented for free. Open source is open source, and a feature won't be added until someone implements it.

By the way there is https://github.com/stevan/p5-plack-debugger that has a completely different architecture on saving the debug related data and retrieve them via JavaScript API.

**Comment by KES777** [comment]
They must = Panels must
under **MUST** I mean 'panel must work in this way' if this feature will be implemented by someone
I do not mean you **MUST** implement it.
Maybe some one see this good idea and implement. Maybe I will notice in future this mine idea and implement.

In other case idea will be forgotten

**Comment by miyagawa** [comment]
> In other case idea will be forgotten

No, you don't need to use the word "must" to make some idea not forgotten. It will be forgotten no matter how it was worded :)
